src/skip-to-content/skip-to-content.scss

```diff
@@ -0,0 +1,32 @@
+// Copyright (c) Bentley Systems, Incorporated. All rights reserved.
+// See LICENSE.md in the project root for license terms and full copyright notice.
+@import '../style/index';
+
+
+@mixin iui-skip-to-content {
+  @include iui-blur;
+  @include iui-focus($offset: 2px, $thickness: 2px);
+
+  border-radius: 99999px;
+  color: t(iui-color-foreground-accessory);
+  padding: ($iui-baseline * 0.75) $iui-m;
+  text-decoration: none;
+  position: absolute;
+  text-align: center;
+  left: 50%;
+  transform: translateX(-50%) translateY(-170%);
+  transition: background-color $iui-speed-fast ease-in-out,
+              transform $iui-speed-fast ease-in-out,
+              box-shadow $iui-speed-fast ease-in-out;
+  
+  &:hover {
+    @include iui-blur($opacity: 3);
+  };
+  
+  &:focus {
```

Comment:
If you want to show a slower exit animation, you can change the transition-duration inside this focus rule